### PR TITLE
Fix typo (paginiation -> pagination)

### PR DESCRIPTION
**Pull request by Bisaloo**
I'm not 100% sure but I can't find any dictionary with an entry for `paginiation` so I'm assuming that's a typo.

### CODE REVIEW

File: codemeta.json [hunk 1]
```diff
@@ -527,7 +527,7 @@
   ],
   "isPartOf": "https://ropensci.org",
   "keywords": ["metadata", "codemeta", "ropensci", "citation", "credit", "linked-data"],
-  "releaseNotes": "https://github.com/ropensci/codemetar/blob/master/NEWS.md",
+  "releaseNotes": "https://github.com/Bisaloo/codemetar/blob/master/NEWS.md",
```
Comment (maelle) [inline codemeta.json]: Can you remove this edit? (I know it's not your fault :joy:)

Comment (Bisaloo) [reply codemeta.json]: I can but it will create it again, won't it?

Comment (Bisaloo) [reply codemeta.json]: Is there a way maybe to skip GitHub actions (similar to `[ci skip]` in the commit message for travis)?

Comment (maelle) [reply codemeta.json]: aaaaaaaaah right. np I'll merge and it'll get updated again.

I suppose one could add the skip based on the commit message, yes, can you please open an issue suggesting that?

**Pull request merged**